connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -322,7 +320,7 @@ private void processTargetStateChanges(Set<String> connectorTargetStateChanges)
 
             // additionally, if the worker is running the connector itself, then we need to
             // request reconfiguration to ensure that config changes while paused take effect
-            if (worker.ownsConnector(connector) && targetState == TargetState.STARTED)
+            if (targetState == TargetState.STARTED)
```

Comment:
`reconfigureConnectorTasksWithRetry()` -> `reconfigureConnector()` early returns if `!worker.isRunning(connector)`, so this is okay to remove the owns check from here. Mainly removed it because it was the only remaining owns call.